pkg/api/v1beta3/types.go

```diff
@@ -1609,3 +1610,19 @@ type SecretList struct {
 
 	Items []Secret `json:"items" description:"items is a list of secret objects"`
 }
+
+// ServerStatus (and ServerStatusList) holds the cluster validation info.
+type ServerStatus struct {
+	Component  string       `json:"component,omitempty"`
+	Health     string       `json:"health,omitempty"`
+	HealthCode probe.Result `json:"healthCode,omitempty"`
```

Comment:
probe.Result probably shouldn't be used here (it can stay in the unversioned types) because then a change to it would break the api.